sdks/python/run_pylint.sh

```diff
@@ -47,6 +47,7 @@ done
 echo "Skipping lint for generated files: $FILES_TO_IGNORE"
 
 echo "Running pylint:"
-pylint apache_beam --ignore-patterns="$FILES_TO_IGNORE"
+# Re-enable the context manager check once https://github.com/PyCQA/pylint/issues/782 is fixed
+pylint apache_beam --ignore-patterns="$FILES_TO_IGNORE" --disable="not-context-manager"
```

Comment:
Add new disables to: https://github.com/apache/beam/blob/master/sdks/python/.pylintrc#L81